### DISCUSSION TITLE
How to choose installation method (Ambient or sidecar) ?

### DISCUSSION THREAD

**Discussion by m8-t** [istio/istio#52178] [Q&A]
Hi there,

my company finally decided to give Istio a shot and I am now evaluating Istio. Right now I am at the point on which mode I want to install. We are running OpenShift 4.14. As far as I understand ambient mode is stable since Istio 1.22, thus some features are still in alpha. I read that ambient and sidecar mode can co-exist. So I guess I should chose the openshift-ambient profile correct? Does this have any drawbacks when I first want to start with sidecar mode? I want to avoid to migrate later-on when istio is adopted and heavily in use. So my plan is to install in ambient mode but work with sidecar mode until all the features are stable and OpenShift is capable of using it properly. Maybe someone here got some experience or advice? :)

Thanks

**Comment by howardjohn** [comment]
There isn't much drawback, but also no real benefit. You will have some latent features (and a Daemonset, ztunnel) deployed without benefit to you, but some risk/cost. The risk and cost should be low, but its not _zero_.

The benefit is almost none though. It should be just as easy to enable ambient at a later date when you are interested in actually using it.
